docs/docs-content/clusters/edge/site-deployment/deploy-private-registry.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_label: "Deploy Cluster with a Private Registry"
+title: "Deploy Cluster with a Private Registry"
+description: "Instructions for creating an Edge Native Cluster Profile"
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["edge"]
+---
+
+Palette Edge supports authentication with private image registries, which allows your cluster to pull images from a private registry during deployment. You can configure your cluster to pull images from a private registry for both cluster creation and cluster repave. To configure a cluster to pull images from a private image registry, provide the registry URL and the credentials needed to authenticate with the registry in the cluster profile. 
+
+:::caution
+If you have specified registry credentials in the `registryCredentials` field in the user data file during the EdgeForge process, the credentials provided in the cluster profile will be ignored.  For more information, refer to [EdgeForge - Build Artifacts](../edgeforge-workflow/palette-canvos.md).
```

Comment:
We can also link to the registry parameter section of the userdata configuration
https://docs.spectrocloud.com/clusters/edge/edge-configuration/installer-reference#external-registry